### ISSUE TITLE
Example and Example Styled - make it just one?

### ISSUE THREAD

**Issue by mithru** [processing/p5.accessibility#115]
#### Nature of issue?

- [x] Existing feature enhancement

#### Feature enhancement details:
Going back to #54 and #101 , on the discussion about making it easy for first timers to try the library, should we try to make it one folder instead of two? Similar to the change in #101 . Currently the only difference seems to be the visibility of the part that's screen-read.

This would be my proposed folder structure:

Examples folder would have just these files (ignore the names - they're just samples to illustrate my point):
```
./index.html
./app.js
./basicShapes.js
./basicLines.js
./anotherExample.js
./yetAnotherExample.js
```

With the following changes:

1. index.html would have the css to display the different modes, but with a comment that would change to ```display:none```

2. index.html would also have just app.js (probably renamed) uncommented, but all other files i.e. basicShapes.js , basicLines.js , et al commented with a prompt to users to uncomment the file they wish to see. 
OR 
just have app.js and a prompt for users to edit ```app.js``` in the code to ```basicShapes.js``` or the file they wish to try.

**Comment by spiray** [comment]
@mithru I think this is definitely better than what we have now. I think we can make it more beginner friendly in the sense that we can make it all in the UI. We can have different tabs or buttons to select different examples. This would obviously be a bigger project and I will leave it to @MathuraMG and @lm-n on what the priority is but this will be worth it in the long run.

**Comment by lm-n** [comment]
Yes this is a great idea and actually it is pretty high up our priority list. Let's talk about UX/UI design for this next week during our call!
